frontend/src/app/features/user-preferences/notifications-settings/page/notifications-settings-page.component.html

```diff
@@ -8,6 +8,7 @@ <h5>{{ text.notifyImmediately.title }}</h5>
   <p>{{ text.notifyImmediately.description }}</p>
 
   <spot-selector-field
+    labelWeight="bold"
```

Comment:
Since bold should be the default, I would not add it to every spot-selector field, but rather change the default in the [component itself](https://github.com/opf/openproject/blob/b99d0381a53aa1ee0e57f24fd95239b28bba3a66/frontend/src/app/spot/components/selector-field/selector-field.component.ts#L40)

Otherwise it would be regular again if someone forgets to add the `labelWeight` attribute.